src/aerovaldb/jsondb/jsonfiledb.py

```diff
@@ -232,6 +234,8 @@ async def _get_version(self, project: str, experiment: str) -> Version:
 
         :return : A Version object.
         """
+        # assert isinstance(project, DecodedStr)
+        # assert isinstance(experiment, DecodedStr)
```

Comment:
why commented? asserts can be kept, might even be removed automatically when run with `python -O`.